web/src/composables/useLogs.ts

```
@@ -1944,97 +2034,109 @@ const useLogs = () => {
-                  index
-                ] < res.data.total) &&
-              queryReq.query.start_time == item[0]
-            ) {
-              searchObj.data.queryResults.partitionDetail.partitionTotal[
-                index
-              ] = res.data.total;
+      try {
+        searchObj.loadingCounter = true;
+        searchObj.data.countErrorMsg = "";
+        queryReq.query.size = 0;
+        delete queryReq.query.from;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)